cves/CVE-2013-0863.yml

```diff
@@ -64,7 +64,9 @@ description_instructions: |
 
   Your target audience is people just like you before you took any course in
   security
-description:
+description: Buffer overflow in the rle_decode functin in libavcode/sanm.c in FFmpg before 1.0.4
```

Comment:
I don't think naming all of the function names, file names, and version numbers is necessary. also function is misspelled 
also  "before 1.0.4 and 1.1.x before 1.1.2" makes no sense to me